mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping

```diff
@@ -1,2 +1,3 @@
 CLASS net/minecraft/class_1547 net/minecraft/client/render/chunk/ChunkBuilder
+	METHOD method_30787 take ()Lnet/minecraft/class_1543;
```

Comment:
take?

Reply:
Could be poll, but this uses `BlockingQueue::take`, not `Queue::poll`

Reply:
i would like @DeadlyMC 's opinion on this one

Reply:
That method gets the next chunk render task for the ChunkWorker. So maybe something like `getNextChunkRenderTask`. Yes its pretty big, but 14 seems to call it `getNextChunkRenderDataTask`